### ISSUE TITLE
What is emb_scale used for?

### ISSUE THREAD

**Issue by Ocelot7777** [ifzhang/FairMOT#111]
id_head = self.emb_scale * F.normalize(id_head)
So why do you use emb_scale?

**Comment by faruknane** [comment]
@Ocelot7777 I have to ask the same question. It throws an error at this line 
![image](https://user-images.githubusercontent.com/37745467/89794555-e202c680-db2f-11ea-9507-16ce5bc53974.png) when opt.nID is 1.
So is there a way that we can avoid using emb_scale and what does it do?

**Comment by zengjie617789** [comment]
It is hardly to train a network on one image~~~

**Comment by nmll** [comment]
I also have the question about that why to use this operation about self.emb_scale? 
id_head = self.emb_scale * F.normalize(id_head)
